README.md

```diff
@@ -442,6 +442,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Kwelo Network](https://www.kwelo.com/network/ip-address) | Locate and get detailed information on IP address | No | Yes | Yes |
 | [LocationIQ](https://locationiq.org/docs/) | Provides forward/reverse geocoding and batch geocoding | `apiKey` | Yes | Yes |
 | [Mapbox](https://www.mapbox.com/developers/) | Create/customize beautiful digital maps | `apiKey` | Yes | Unknown |
+| [MetaURLtags](https://promptapi.com/marketplace/) | Scrapes, parses and retrieves META tags of any URL | `apiKey` | Yes | Unknown |
```

Comment:
Looks like a marketplace of APIs. Is it the URL you wanna add?

Reply:
Ganhar dinheiro baixando o Kwai!https://s.kwai.app/s/oenFdtwS